201-front-door-premium-storage-blobs-private-link/metadata.json

```diff
@@ -0,0 +1,12 @@
+{

```

Comment:
Are these going to be referenced in docs - wondering if we need the docOwner property on them?

Reply:
Sure - I have added in.